docs/docs/en/guide/datasource/databend.md

```diff
@@ -0,0 +1,21 @@
+# Databend
+
+![Databend Datasource](../../../../img/new_ui/dev/datasource/Databend.png)
+
+## Datasource Parameters
+
+|     **Datasource**      |                       **Description**                       |
+|-------------------------|-------------------------------------------------------------|
+| Datasource              | Select DATABEND.                                            |
+| Datasource Name         | Enter the name of the datasource.                           |
+| Description             | Enter a description of the datasource.                      |
+| IP/Host Name            | Enter the DATABEND service IP.                              |
+| Port                    | Enter the DATABEND service port.                            |
+| Username                | Set the username for DATABEND connection.                   |
+| Password                | Set the password for DATABEND connection.                   |
+| Database Name           | Enter the database name of the DATABEND connection.         |
+| jdbc connect parameters | Parameter settings for DATABEND connection, in JSON format. |
+
+## Native Supported
+
+Yes, could use this datasource by default.
```

Comment:
Chinese and English version is different?

Reply:
should also add new docs in https://github.com/apache/dolphinscheduler/blob/c0126b7f30ff335aa1cfd62508b041084fb090d5/docs/configs/docsdev.js#L1011-L1014 and https://github.com/apache/dolphinscheduler/blob/c0126b7f30ff335aa1cfd62508b041084fb090d5/docs/configs/docsdev.js#L330-L333

Reply:
Add this line for `databend` but where is the content of `/en-us/docs/dev/user_doc/guide/datasource/ssh.html`?

Reply:
it will auto-generated during website build, from markdown to html